.circleci/config.yml

```diff
@@ -27,4 +27,4 @@ workflows:
       - build:
           matrix:
             parameters:
-              jdk-version: ["8", "9"]
+              jdk-version: ["11"]
```

Comment:
Let's also change the base image to `cimg/openjdk` as the one we are using right now is deprecated.

Reply:
done